test/supports/volume_local.py

```diff
@@ -0,0 +1,16 @@
+# Copyright Modal Labs 2022
+from __future__ import annotations
```

Comment:
I'd remove this future import as it has a lot of weird side effects. Would be nice to remove from other places where we use it too, but that's unrelated to this PR :D